### ISSUE TITLE
Transfer items between scrapbooks

### ISSUE THREAD

**Issue by hermann-san** [danny0838/webscrapbook#283]
Hi,  I've got webscrapbook on Windows 10 with a single scrapbook, but now I'd like to have multiple scrapbooks for my backend server. So I did the configuration and things are fine with multi scrapbook. I can see 2 scrapbooks in the sidebar and they are empty. However, I don't know how to get the files from my old scrapbook into the sidebar. with multi scrapbooks feature. I did configure it in the config.ini with the existing data, but the files don't show up  in the sidebar. 

2022-1 is the existing scrapbook with data, but the data doesn't show up in the sidebar.
Is the single scrapbook data structure compatible with multiple scrapbook data structure?  If not, how can I move my data to the multi scrapbook file directory?

Thanks a lot for you hints.

[book ""]
name = 2022-1
top_dir = 2022-1
data_dir = 
tree_dir = .wsb/tree
index = .wsb/tree/map.html
no_tree = false

[book "2022-2"]
name = 2022-2
top_dir = 2022-2 
data_dir = 
tree_dir = .wsb/tree
index = .wsb/tree/map.html
no_tree = false

**UPDATE:**
I've figured it out. 
1. config.ini file in \.wsb need to be configured for multi scrapbook and existing scrapbook need to be included in the config (2022-1 in the  above example)
2. Existing scrapbook HTZ Files  need to be moved from \data to \
3. *js files need to be moved from \.wsb\tree to \tree
After a refresh in the Browser sidebar, the files are there.

**Comment by danny0838** [comment]
Simply use the `Copy to...` context menu command from the sidebar. See [doc](https://github.com/danny0838/webscrapbook/wiki/Backend#transfer-items-between-scrapbooks) for more details.

**Comment by hermann-san** [comment]
the problem is  that I first have to get the existing scrapbook (with a lot of data) into the sidebar.  Copy to works only if the scrapbooks are in the sidebar and that was only possible with the procedure I've described above.

**Comment by danny0838** [comment]
I don't get you. You said you have got it work in the updated message, didn't you? Is this issue still active now?

**Comment by hermann-san** [comment]
the scrapbook in the sidebar which I meant is the one that I initially had. Without multi scrapbook. So about 4 month ago I setup webscrapbook with a single book. That worked fine. 
Then I've moved to multi scrapbook and wanted to include that existing scrapbook  in the multi scrapbook config.  So I've created a multi scrapbook config and thought I could simply add the existing scrapbook into the config.ini . That didn't work. 
So I've used by workaround mentioned above and things worked fine (in the first post).  

For me the problem is solved. 

I just mentioned my solution in the case someone else is running into the problem. 
Here is a small diagram. Hopefully it explains it better. 

![grafik](https://user-images.githubusercontent.com/7679654/179403580-b919fb9d-2fc6-4c2f-894c-627dd2bfd181.png)
